### ISSUE TITLE
overlay crashes when applying or disabling mods (tesla) on 2.02

### ISSUE THREAD

**Issue by cagerhager** [nadrino/SimpleModManager#13]
It works great with 1.6, but other versions after that will crash when using overlay. The mod manager program works through homebrew. I am on Atmosphere 13, Firmware 1.0.4. and have it installed on emmu nand.

**Comment by nadrino** [comment]
Thanks a lot for repporting this here !

Alright, so there is indeed a crash when we try to disable any mod. I'm investigating it, keep you in touch.

**Comment by nadrino** [comment]
Ok, this is now fixed! I'm not exactly sure of how it worked on 1.6.0 but it will be fixed in the next release 2.0.2b.
This issue was comming from the way I updated the status bar of each mod. I was actually rebuilding the whole UI each time the user made an operation (apply/disable), and it was making the overlay crash. Now SMM overlay will only override the status bar, which is quicker and the focus isn't lost anymore !
